app/src/main/java/io/apicurio/registry/storage/impl/AbstractMapRegistryStorage.java

```diff
@@ -133,11 +133,9 @@ private boolean filterSearchResult(String search, String artifactId, SearchOver
             switch (searchOver) {
                 case name:
                 case description:
+                case labels:
                     String value = getLatestContentMap(artifactId, ArtifactStateExt.ACTIVE_STATES).get(searchOver.name());
                     return value != null && value.contains(search);
```

Comment:
I think this allows for false positives on label searches, but it's OK for this release.  We can wait for someone to complain...